syft/workers/base.py

```diff
@@ -110,6 +103,7 @@ def __init__(
         self.hook = hook
 
         self.object_store = ObjectStore(owner=self)
+        self.message_handler = BaseMessageHandler(self.object_store, self)
```

Comment:
Q: can't we pass only the ```self``` and get the object store from it?

Reply:
Yeah, we could. This `MessageHandler` in particular has a dependency on the worker, but in general they should only depend on the `ObjectStore`. If we find that more of them have dependencies on the worker we can consolidate these params, but I'm still trying to encourage future message handlers to be relatively self-contained.